.github/workflows/config/validation_config.json

```diff
@@ -51,32 +51,32 @@
                 "fedora:38"
             ],
             "ubuntu:22.04": 
-            { 
-                "libstdcpp_package": "libstdc++-11-dev",
+            {
+                "libcdev_package": "libc6-dev",
```

Comment:
With this change, having a C++ toolchain installed separately from CUDA-Q is no longer necessary - the installer includes a complete C++ toolchain. However, we still depend on the C-libraries on the system, including the C development headers. Docker images (i.e. the environment used for validation) do usually have a more minimal set of packages installed than is usual on most operating systems. Specifically, libc6-dev is listed because the C development headers are not installed in the image used for validation. This remains a dependency, see the RFC and slides for more detail.